modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/upgrade/v10_0_0/CommerceOrderItemUpgradeProcess.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.internal.upgrade.v10_0_0;
+
+import com.liferay.commerce.model.impl.CommerceOrderItemModelImpl;
+import com.liferay.petra.string.StringBundler;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+
+/**
+ * @author Stefano Motta
+ */
+public class CommerceOrderItemUpgradeProcess extends UpgradeProcess {
+
+	@Override
+	public void doUpgrade() throws Exception {
+		if (hasColumn(
+				CommerceOrderItemModelImpl.TABLE_NAME, "decimalQuantity")) {
+
+			_alterColumnType(
```

Comment:
why?
we already have a decimal quantity column... we just need to rename that one to quantity and remove the decimalQuantity column

Reply:
Done in this way because is how it's specified in the requirements